UnitTests/Utilities/GeometryModelRenderer/src/BinGeometryModelRenderer.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+	Copyright (c) 2021 Technical University of Munich
+	Chair of Computational Modeling and Simulation.
+
+	TUM Open Infra Platform is free software; you can redistribute it and/or modify
+	it under the terms of the GNU General Public License Version 3
+	as published by the Free Software Foundation.
+
+	TUM Open Infra Platform is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "OffGeometryModelRenderer.h"
+
+OffGeometryModelRenderer::OffGeometryModelRenderer(const buw::ReferenceCounted<buw::IRenderSystem>& renderSystem) 
+	: GeometryModelRenderer::GeometryModelRenderer(renderSystem)
+{
+	offGeometryEffect_ = buw::makeReferenceCounted<oip::OffGeometryEffect>(
```

Comment:
If you plan to use this for point clouds you should also use the PointCloudEffect
(In the whole file not just this section)

Reply:
Yes, I will change it